suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -193,6 +197,15 @@ export const selectDeviceEnabledDiscoveryNetworkSymbols = memoizeWithArgs(
     { size: 2 },
 );
 
+export const selectTokenDefinitionsEnabledNetworks = memoize(
```

Comment:
Looking at the selector name, it's not obvious what is being selected... Token definitions or enabled networks? 🤔

Reply:
I just kept the name used elsewhere

Reply:
It's confusing but if it's used elsewhere... 😏